### DISCUSSION TITLE
Checking getValue

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4168] [Q&A] [answered]
I write the text "0" into the structure, and when checking the condition, the condition does not work, although the output works properly

Code example
```c
item.Items = ini.GetValue(defIdChar, "items", "noone");
```

Here the condition is triggered only else, although numbers are written in json, and 0 is no exception
```c
if (dataBegin->Items == "0")
{
    returnItems = "Нет в наличии";
}
else
{
    returnItems = dataBegin->Items;
}
```
What could be the reason?

**Comment by nlohmann** [comment, marked as answer]
This library does not have a GetValue function. You may have mistaken the repository.
